### PR TITLE
Massive Changes

### PR DESCRIPTION
**Pull request by TNTzx**
- changed schema
- removed old code
- created new structures conforming to the new schema
- currently nothing relating to express at the moment

(note: i'm stupid and committed some with my other account lmao)

### CODE REVIEW

File: package.json
Comment (naoei) [inline package.json]: These shouldn't link to your fork

File: src/structs/artist.ts
Comment (naoei) [inline src/structs/artist.ts]:
```suggestion
import { Tag } from "./tag"
```

File: src/structs/artistRequest.ts
Comment (naoei) [inline src/structs/artistRequest.ts:45]: This implies that we're going to have a user system, which I don't think we'll ever have. This site should primarily be anonymous.

File: src/structs/artistRequest.ts
Comment (naoei) [inline src/structs/artistRequest.ts:5]:
```suggestion
import { Tag } from "./tag"
import { Artist } from "./artist"
```

File: src/structs/artist.ts
Comment (naoei) [inline src/structs/artist.ts:23]: I feel like the default values would read better if we had something along the lines of.

```ts
export class Song {
    audioPath: string;
    title: string;
    romanizedTitle?: string;

    constructor(data: Song) {
        this.audioPath = data.audioPath
        this.title = data.title ?? "Unnamed Song"
        this.romanizedTitle = data.romanizedTitle
    }
}

new Song({
    audioPath: "",
}); // { audioPath: "", title: "Unnamed Song", romanizedTitle: undefined }
```

**Review by naoei — CHANGES_REQUESTED** [review]
Overall this PR removes everything that I've done before, which isn't exactly something I would merge given normal circumstances, but since we're starting from scratch again it's okay.

In the end, I'd like to have the same features as before (GraphQL, and REST API).

(Just as a side note, I'd highly recommend you clean up your commit history here. I'm gonna hold off from merging this until that's also fixed.)

File: src/structs/artistRequest.ts
Comment (TNTzx) [reply src/structs/artistRequest.ts:45]: hmm i was thinking this would be used for the bot where `requestorUserId` would be the discord id
though i guess i'm just conflicted if that info should be stored in the bot's db or in the site's db

**Comment by TNTzx** [comment]
gonna cancel the pr since it has too much issues and we'd probably restart anyway

**Pull request closed without merging**
